vmm/src/config.rs

```diff
@@ -525,6 +535,25 @@ impl CpusConfig {
                     })
                     .collect()
             });
+        let features_list = parser
+            .convert::<StringList>("features")
+            .map_err(Error::ParseCpus)?
+            .unwrap_or_default();
+        // Some ugliness here as the features being checked might be disabled
+        // at compile time causing the below allow and the need to specify the
+        // ref type in the match.
+        #[allow(unused_mut)]
```

Comment:
Yeah unfortunately that's because we support only the AMX feature. This might go away if we add more features.
Or you could have an `if/else` statement defining `features`.

Reply:
I ended up keeping it as is, my expectation is that moving the `kvm_hyperv` option into this will be soonish anyway making this no longer an issue.

Reply:
Sounds good! Can you add this to the comment? I mean specify this should be simplified when moving `kvm_hyperv` under `CpuFeatures`.

Reply:
Done.